client/app/resources/i18n/domain/Messages_cs_CZ.xml

```diff
@@ -589,7 +589,8 @@ Ve výchozím nastavení jsou zveřejněny v WHOIS databázi, která je veřejn
    <translation id="domain_configuration_dns_entry_add_publickey_revoke" qtlid="192396">Zrušit veřejný klíč</translation>
    <translation id="domain_configuration_dns_entry_add_servicetypes" qtlid="192409">Typy služeb</translation>
    <translation id="domain_configuration_dns_entry_add_servicetypes_all" qtlid="69173">Vše</translation>
-   <translation id="domain_configuration_dns_entry_add_servicetypes_email" qtlid="192422">e-mail</translation>
+   <translation id="domain_configuration_dns_entry_add_servicetypes_email" qtlid="192422">E-mail</translation>
```

Comment:
No need to update these file, translations are managed by OVH ;)

You should update the `fr_FR` file only ;)